Packs/RedCanary/Integrations/RedCanary/RedCanary.py

```diff
@@ -206,14 +205,13 @@ def process_timeline(detection_id):
     ips = []
     processes = []
     for activity in res:
-        if activity["type"] != "activity_timelines.ActivityOccurred":
+        if activity.get('type') not in ["activity_timelines.LatestIndicationSeen", "activity_timelines.EventActivityOccurred"]:
```

Comment:
what happened with ActivityOccurred? never relevant anymore?
where are these two values from?

Reply:
I don't know. I don't have access to the product. 
But what I used is from a customer response (deleted secrets, of course)
It is from his response, so I could replicate the issue via unittest.
I didn't delete the old value cause I can't be sure it is irrelevant, this way we're handling both cases